meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb

```diff
@@ -0,0 +1,19 @@
+DESCRIPTION = "Very fast, header only, C++ logging library."
+HOMEPAGE = "https://github.com/gabime/spdlog/wiki"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+PR="r0"
+SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;tag=v${PV}"
```

Comment:
No tags please. use hardcoded git SHA, tags are floating and dont work in NO_NETWORK situations

Reply:
done